toolkits/kaldi/gigaspeech_data_prep.sh

```diff
@@ -80,14 +153,18 @@ if [ $stage -le 3 ]; then
   echo "$0: Split data to train, dev and test"
   # Split data to train, dev and test.
   [ ! -f $corpus_dir/utt2subsets ] &&\
-    echo "$0: Error: No such file $corpus_dir/utt2subsets!" && exit 1
-  for subset in ${!subsets[*]}; do
+    echo "$0: No such file $corpus_dir/utt2subsets!" && exit 1
+  for label in $train_subset DEV TEST; do
```

Comment:
it is still not clear to me that if we should pair entire DEV(around 3000 utts) & TEST(around 8000 utts) with XS for pipeline coding/debugging purposes, decoding DEV and TEST might be slow for this usage.  We can leave this as an open question for now.

Reply:
Makes sense, but I also don't want to add to much complexity in the subsets. Let's leave this for now.